src/lib.rs

```diff
@@ -115,6 +115,7 @@ pub struct PageHeadingItemCompat {
     pub number: Option<String>,
     pub title: Option<String>,
     pub path: Option<String>,
+    pub bury: bool,
```

Comment:
Why do we need this field in "page-headings processor" struct? This is a sitemap feature.

Reply:
okay. got it. closing the PR now.